### ISSUE TITLE
[Bug]: Epoch limited to 1000 for textual inversions

### ISSUE THREAD

**Issue by rockerBOO** [civitai/civitai#202]
### What happened?

Inputting epoch of 1478 into the epoch of a textual inversion. It reverts it to 1000. 

### Steps to reproduce the problem

Enter 1478 into the epoch of a textual inversion

### What should have happened?

Should keep the epoch of 1478.

### What platforms do you use to access the site?

Linux

### What browsers do you use to access the site?

Mozilla Firefox

### Additional information, context and logs

There may be a sane limit for this field, but for my case it's too low. If you do batch gradient descent (of the whole training set) each step would be a full epoch. This contributed to the epoch (1478) being large at 4,476 steps, with steps per epoch starting at 1 then to 3 steps per epoch later.

**Comment by manuelurenah** [comment]
Hey @rockerBOO this has already been implemented since yesterday afternoon, sorry for the late reply. Feel free to bump those epoch numbers up now 🚀
